doc/Core.xml

```diff
@@ -6720,6 +6720,84 @@ Event description:
 </tt:MessageDescription>
 ]]></programlisting>
       </section>
+      <section>
+        <title>Sensor Data Event Topics</title>
+        <para>ONVIF defines the following topics related to sensor data:</para>
+        <itemizedlist>
+          <listitem>
+            <para>tns1:Device/Sensor/[SensorType] - Generic sensor event topic</para>
+          </listitem>
+          <listitem>
+            <para>tns1:Device/Sensor/temperature/High - Temperature too high event</para>
```

Comment:
I believe all topics should be Pascal cased so topics are standard. So it would be better to change them to
```
tns1:Device/Sensor/Temperature/High
```